Routes/post.js

```diff
@@ -51,26 +51,58 @@ router.patch('/', function(req, res) {
         query = 'UPDATE post SET title = ?, body = ? WHERE post_id = ?';
         values = [req.body.title, req.body.body, req.body.post_id];
 
-    // We Patching only Title
+        // We Patching only Title
     } else if (req.body.title) {
         query = 'UPDATE post SET title = ? WHERE post_id = ?';
         values = [req.body.title, req.body.post_id];
 
-    // We Patching only Body
+        // We Patching only Body
     } else if (req.body.body) {
         query = 'UPDATE post SET body = ? WHERE post_id = ?';
         values = [req.body.body, req.body.post_id];
     }
 
-   // Execute based on the different scenario
-   pool.query(query, values, (error, results) => {
+    // Execute based on the different scenario
+    pool.query(query, values, (error, results) => {
+        if (error) {
+            res.status(500).json({ error: 'Error updating the post.' });
+        } else {
+            res.status(200).json({ message: 'Post updated successfully.' });
+        }
+    })
+});
+
+//Handling the edit post 
+router.delete('/', function (req, res) {
```

Comment:
I think we should change this DELETE route from "/" to "/comment" to be more specific about what's being deleted.